src/components/AttachmentModal.js

```diff
@@ -145,6 +149,15 @@ class AttachmentModal extends PureComponent {
             : [styles.imageModalImageCenterContainer, styles.p5];
 
         const {fileName, fileExtension} = this.splitExtensionFromFileName();
+
+        let headerTitle = '';
+        if (this.props.isProfilePicture) {
+            headerTitle = this.props.displayName;
+        } else {
+            headerTitle = this.props.isUploadingAttachment
+                ? this.props.translate('reportActionCompose.sendAttachment')
+                : this.props.translate('common.attachment');
+        }
```

Comment:
I think it would be better to take headerTitle prop instead and use its value which defaults to  `this.props.translate('common.attachment')`